Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\BqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-BqJob" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+        $folder = Get-Location
+        $folder = $folder.ToString()
+        $filename = "$folder\classics.csv"
+        $table = New-BqTable -Dataset $test_Set "table_$r"
+        New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
+            New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
+            Add-BqTabledata $filename CSV -SkipLeadingRows 1
+        $table | Add-BqTabledata $filename CSV -SkipLeadingRows 1
+    }
+
+    It "should list jobs from the past 6 months" {
+        $jobs = Get-BqJob
+        $jobs.Count | Should BeGreaterThan 1
+    }
+
+    #TODO(ahandley): When Start- and Stop-BqJob are written, add in tests for AllUsers and State
+
+    It "should get specific job via pipeline" {
+        $jobs = Get-BqJob
+        $job = $jobs[0] 
+        $return = $job | Get-BqJob
+        $return.JobReference.JobId | Should Be $job.JobReference.JobId
+    }
+
+    It "should get specific job via parameter" {
+        $jobs = Get-BqJob
+        $job = $jobs[0] 
+        $return = Get-BqJob -JobId $job.JobReference.JobId
+        $return.JobReference.JobId | Should Be $job.JobReference.JobId
+    }
+
```

Comment:
Add a test like:

$jobs = Get-BqJob -Project $someOtherProject
$job = $jobs[0]
$return = $job | Get-BqJob
$return.JobReference.JobId | Should Be $job.JobReference.JobId
$return.JobReference.ProjectId | Should Be $job.JobReference.ProjectId